packages/airnode-protocol-v1/contracts/monetization/RequesterAuthorizerWhitelisterWithTokenDeposit.sol

```diff
@@ -59,6 +66,21 @@ contract RequesterAuthorizerWhitelisterWithTokenDeposit is
         )
     {}
 
+    /// @notice Called by the manager to set the withdrawal lead time
+    /// @param _withdrawalLeadTime Withdrawal lead time
+    function setWithdrawalLeadTime(uint256 _withdrawalLeadTime)
+        external
+        override
+        onlyMaintainerOrManager
+    {
+        require(
+            _withdrawalLeadTime <= 30 days,
```

Comment:
This is only there to prevent `withdrawalLeadTime` from being set to 10 years and someone accidentally signaling withdrawal intent, locking the tokens practically forever. We will probably set it at something like 1 day if we ever use it.

Reply:
referring to this

> Implemented a lead time requirement to prevent front-running the block transaction, but kept it disabled by default assuming we will never need it. The lead time is capped at **1 week** to prevent its abuse.

Reply:
Right, I don't know what I was thinking